core/src/test/java/org/apache/brooklyn/core/mgmt/persist/FileBasedStoreObjectAccessorWriterTest.java

```diff
@@ -61,6 +76,30 @@ public void testFilePermissions600() throws Exception {
         FileBasedObjectStoreTest.assertFilePermission600(file);
     }
 
+    // Fails ~3 times on 5000 runs in Virtualbox (Ubuntu Xenial).
```

Comment:
Is it worth a  comment here that the test is intended to illustrate the problem, which is now addressed by the change in `RebindTestUtils`, so though this is @Broken it isn't a test that someone needs to fix?